src/main/kotlin/org/kamiblue/client/module/modules/combat/CombatSetting.kt

```diff
@@ -96,10 +95,10 @@ internal object CombatSetting : Module(
     private var overrideRange = range.value
     private var paused = false
     private val resumeTimer = TickTimer(TimeUnit.SECONDS)
-    private val jobMap = hashMapOf<(SafeClientEvent) -> Unit, Job?>(
-        { it: SafeClientEvent -> it.updateTarget() } to null,
-        { it: SafeClientEvent -> it.updatePlacingList() } to null,
-        { it: SafeClientEvent -> it.updateCrystalList() } to null
+    private val taskList = arrayListOf<(SafeClientEvent) -> Unit>(
+        { it.updateTarget() },
+        { it.updatePlacingList() },
+        { it.updateCrystalList() }
     )
```

Comment:
This is for preventing starting new calculations if the previous one wasn't done yet

Reply:
I understand that, but this is in support of the behavioral changes I made.